01/app.js

```diff
@@ -1,2 +1,12 @@
 const randomNumber = Math.round(Math.random() * 20);
+const numbersToLog = [];
 console.log(randomNumber);
+
+if (randomNumber>5) {
+    for(let i=5; i<=randomNumber; i++) {
+        numbersToLog.push(i);
```

Comment:
Jeśli mielibyśmy się trzymać sztywno treści zadania to należało już w tym miejscu wyświetlić liczbę :)